src/networking/httpService.ts

```diff
@@ -1,4 +1,4 @@
-const baseUrl = 'https://jsonplaceholder.typicode.com'
+const baseUrl = 'https://node-training-1ym5.onrender.com'
```

Comment:
This can be added to a `.env` file, what do you think?

https://github.com/lugg/react-native-config